### ISSUE TITLE
UX improvements

### ISSUE THREAD

**Issue by neuhausj** [FrSkyRC/ETHOS-Feedback-Community#4507]
I have a X18SE and after using it for a while, I noticed some potential UX improvements:

1. Display the key that has been pressed when using the keyboard. This would help confirm the right key has been pressed (same as on a phone)
2. When opening a menu, the first option should be highlighted (instead of no option currently). This is specially useful when using the wheel.
3. Allow the creation of templates to ease the creation of new models (all widgets/screens + all config options)
4. Allow customisation of the top bar (ex: receiver voltage & RSSI)
5. When turning the wheel backward, allow to go back at the end of the list. This is already working as intended in Mixes, Trims, but not in Outputs or RF system. 
6. When editing a text using the touchscreen, it is not possible to place the cursor before the first letter (only after the first letter is possible). Also, an animation showing where the cursor is while being dragged would be helpful.

On top of that, here is a bug in the latest version (1.5.15)
**BUG**: on vars, when adding 8 actions or more, the last line is not displayed correctly anymore (after going back and reopening the variable). Either the button "New action" is not displayed, or the last line is not displayed.

**Comment by neuhausj** [comment]
Is there an update to this?
1-2 are direct usability improvements
3-4 would be nice to have, but clearly not high priority. For templates: already locking edition of a model could be an improvement (only cloning possible)
5-6 are bugs

Thanks for the support !
